aws-python/__main__.py

```diff
@@ -1,10 +1,6 @@
-"""An AWS Python Pulumi program"""
-
 import pulumi
-from pulumi_aws import s3
 
-# Create an AWS resource (S3 Bucket)
-bucket = s3.BucketV2('my-bucket')
+import infra
```

Comment:
Moves resource creation to a separate module, which is consistent with the change for `aws-typescript`. See https://github.com/pulumi/templates/pull/863#discussion_r1889302372.